sushi-config.yaml

```diff
@@ -30,3 +30,5 @@ dependencies:
     who.ddcc: dev
     who.ips-pilgrimage: dev
     
+# parameters:
+#   apply-version: true
```

Comment:
@costateixeira  please also upgrade the version to 1.1.1 with this change and describe change in publication-request json